app/renderer/lib/client-frameworks/java.js

```diff
@@ -72,31 +63,26 @@ ${this.indent(code, 4)}
 `;
   }
 
-  codeFor_executeScript (/*varNameIgnore, varIndexIgnore, args*/) {
-    return `/* TODO implement executeScript */`;
-  }
-
   codeFor_findAndAssign (strategy, locator, localVar, isArray) {
     let suffixMap = {
-      xpath: 'XPath',
-      'accessibility id': 'AccessibilityId',
-      'id': 'Id',
-      'class name': 'ClassName',
-      'name': 'Name',
-      '-android uiautomator': 'AndroidUIAutomator',
-      '-android datamatcher': 'AndroidDataMatcher',
-      '-android viewtag': 'AndroidViewTag',
-      '-ios predicate string': 'IosNsPredicate',
-      '-ios class chain': 'IosClassChain',
+      'xpath': 'xpath',
```

Comment:
do we really need to quote object keys if these don't contain special characters?

Reply:
I just added them for consistency, but I'll remove them